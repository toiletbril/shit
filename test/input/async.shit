echo &
echo 1 & echo 1
echo 1 | cat &