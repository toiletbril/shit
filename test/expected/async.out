[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "echo", Async]
  [CompoundListCondition, None]
    [SimpleCommand "echo 1", Async]
  [CompoundListCondition, None]
    [SimpleCommand "echo 1"]
  [CompoundListCondition, None]
    [Pipeline, Async]
      [SimpleCommand "echo 1"]
      [SimpleCommand "cat"]
[Escape Map
  
]

1
1
[Code 0]
1
